t/lib/Bakesale/Context.pm

```diff
@@ -51,4 +51,25 @@ package Bakesale::Context::System {
   with 'Ix::Context';
 }
 
+package Bakesale::Context::NoAuth {
+  use Moose;
+
+  sub code { 401 }
+
+  with 'Ix::Context::Error';
+}
+
+package Bakesale::Context::BadAuth {
+  use Moose;
+  use experimental qw(lexical_subs signatures postderef);
+
+  sub code { 401 }
+
+  sub modify_response ($self, $resp) {
+    $resp->body('{"error":"bad auth"}');
+  }
+
```

Comment:
This case is if we want to specify a specific body, or more likely, nuke the user's cookies